src/qforte/abc/algorithm.py

```diff
@@ -249,6 +250,8 @@ def fill_pool(self):
         """ This function populates an operator pool with SQOperator objects.
         """
 
+        from qforte import sq_op_find_symmetry
```

Comment:
All `import` statements should be moved to the top of the file.

Reply:
If I move the import statements to the top of the file wouldn't sq_op_find_symmetry be imported even in situations where we do not needed? I thought that placing it inside the particular function that requires it would help with memory and, perhaps, speeding up the code.

Reply:
Yes, it may  imported even when not needed, but that's not a problem. Performance impact of `sq_op_find_symmetry` is likely negligible.